src/app/components/LogoutHeader/index.tsx

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
 import { Image, TouchableOpacity } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import { wipeLoginData } from '@localStore';
+import { useDispatch } from 'react-redux';
+import actionCreators from '@redux/login/actions';
 import styles from './styles';
 
 const LogoutHeader = () => {
+  const navigation = useNavigation();
+  const dispatch = useDispatch();
+  const logOut = () => {
+    wipeLoginData();
+    dispatch(actionCreators.clearLoginData());
+    navigation.navigate('Login');
```

Comment:
you should reset navigation, not navigate because you can go back on navigate actions

Reply:
Done!

Reply:
(Anyway, I can tell ... when I logged out, the Back button does not navigate to another view)